packages/features/settings/TimezoneChangeDialog.tsx

```diff
@@ -28,7 +28,10 @@ const TimezoneChangeDialogContent = () => {
   const formattedCurrentTz = CURRENT_TIMEZONE.replace("_", " ");
 
   const onMutationSuccess = async () => {
-    showToast(t("updated_timezone_to", { formattedCurrentTz }), "success");
+    showToast(
+      t("updated_timezone_to", { formattedCurrentTz, interpolation: { escapeValue: false } }),
+      "success"
```

Comment:
Fixes this

![image](https://github.com/user-attachments/assets/fd2c363d-f634-4145-8eb5-0df283545fc3)